tests/director/gen-examples.sh

```diff
@@ -68,6 +69,10 @@ docker run --rm --network=${NETWORK} \
     -e MIGRATION_PATH="director" \
     ${MIGRATOR_IMG_NAME}
 
+
+echo -e "${GREEN}Seeding the db...${NC}"
+PGPASSWORD=${DB_PWD} psql -h ${POSTGRES_CONTAINER} -U ${DB_USER} -f <(cat seeds/tenants/*.sql) ${DB_NAME}
```

Comment:
Yet another path that should be `seeds/director/*.sql`. Also `-f` expects file path.